### PR TITLE
Wikimed custom app build

### PR DESCRIPTION
**Pull request by BPerlakiH**
Implementing: [#521](https://github.com/kiwix/apple/issues/521)
and: [#47](https://github.com/kiwix/kiwix-apple-custom/issues/47)

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ on:
     types: [published]
     branches:
       - main
```
Comment (BPerlakiH) [inline .github/workflows/cd.yml:7]: This was only to be able to run it without merging

File: .github/workflows/cd.yml [hunk 2]
```diff
@@ -29,6 +30,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           path: custom
+          ref: wikimed
```
Comment (BPerlakiH) [inline .github/workflows/cd.yml:33]: same here: only to be able to run it without merging to main

File: wikimed/info.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+    "about_app_url": "https://www.kiwix.org/support",
+    "about_text": "Offline Medical Wikipedia",
+    "app_name": "WikiMed",
+    "app_store_id": "id1281693200",
+    "bundle_id": "org.kiwix.wikimed",
```
Comment (BPerlakiH) [inline wikimed/info.json:6]: This was the already set up bundle_id for the app in AppStore, so I re-used it.

File: wikimed/info.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+    "about_app_url": "https://www.kiwix.org/support",
+    "about_text": "Offline Medical Wikipedia",
```
Comment (BPerlakiH) [inline wikimed/info.json:3]: I wasn't sure about this part, but now it's easy to edit it.

Comment (rgaudin) [reply wikimed/info.json:3]: Ah right we don't have this on android.

**Review by rgaudin — APPROVED** [review]
LGTM (code changes). Please remove the non-`main` related stuff. Let's ping @kelson42 on slack so he takes a look at branding properties in JSON (or redirect to the person in charge)

**Pull request merged**